extensions/amp-list/0.1/amp-list.js

```diff
@@ -385,6 +386,7 @@ export class AmpList extends AMP.BaseElement {
    */
   ssrTemplate_(refresh) {
     let request;
+    assertHttpsUrl(this.element.getAttribute('src'), this.element);
```

Comment:
Why is this line necessary? In what cases is the URL not HTTPS?

Reply:
the non ssr code path made the check so I added it but src is not bindable and forbidden in the amp4email spec, where if allowed would be an issue so I'll remove. Thanks

Reply:
I think amp-bind also enforces that src is HTTPS-only FYI.